client/blocks/reader-post-card/style.scss

```diff
@@ -350,15 +350,21 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	font-weight: 100;
 	margin-top: 9px;
 	word-break: break-word;
-	overflow: hidden;
-	max-height: 15px * 1.6 * 3;
 	position: relative;
+}
 
-	&::before {
-		@include long-content-fade( $size: 15px * 1.6 * 5 );
-		top: inherit;
-		height: 15px * 1.6;
-	}
+// If we're not showing the entire excerpt, clamp to 3 lines
+.reader-post-card.card:not(.is-showing-entire-excerpt) {
+	.reader-post-card__excerpt {
+    	overflow: hidden;
+		max-height: 15px * 1.6 * 3;
```

Comment:
I like it just because it's explanatory and less magical. `72px` wouldn't tell me _why_ 72.